tests/e2e/default/tasks/tasks-breadcrumbs-config.js

```diff
@@ -0,0 +1,50 @@
+const oneDay = 24 * 60 * 60 * 1000;
+const isFormArraySubmittedInWindow = (reports, formArray, start, end, count) => {
+  end = end || start + 10 * oneDay;
+  let found = false;
+  let reportCount = 0;
+  reports.forEach(function (report) {
+    if (formArray.includes(report.form)) {
+      if (report.reported_date >= start && report.reported_date <= end) {
+        found = true;
+        if (count) {
+          reportCount++;
+        }
+      }
+    }
+  });
+
+  return count ? reportCount >= count : found;
+};
+
+module.exports = [
+  {
+    name: 'person_create',
+    icon: 'icon-person',
+    title: 'person_create',
+    appliesTo: 'contacts',
+    appliesToType: ['person'],
```

Comment:
It would be exciting to have a general task config file where we can create different types of tasks. For example, it would be helpful to create a task that applies to a place and other that applies to a report, to enhance the scenarios in tasks-breadcrumbs. What do you think?

Reply:
I'm not sure that I understand the question very well but I would say, as explained in the issues https://github.com/medic/cht-core/pull/7732 https://github.com/medic/cht-core/pull/7757, it should not matter in this case if the task applies to a place or to a person, the most important is their linege, and the place of the connected user, to display the breadcrumbs as requested.

Reply:
Ok, so I understand those different scenarios will be redundant. Perfect.